packages/sdk38/src/testutils.spec.ts

```diff
@@ -19,6 +19,9 @@ export const tendermintAddressMatcher = /^[0-9A-F]{40}$/;
 export const tendermintShortHashMatcher = /^[0-9a-f]{40}$/;
 export const dateTimeStampMatcher = /^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(?:\.[0-9]+)?Z$/;
 export const semverMatcher = /^[0-9]+\.[0-9]+\.[0-9]+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$/;
+/** @see https://rgxdb.com/r/1NUN74O6 */
+export const base64Matcher = /^(?:[a-zA-Z0-9+/]{4})*(?:|(?:[a-zA-Z0-9+/]{3}=)|(?:[a-zA-Z0-9+/]{2}==)|(?:[a-zA-Z0-9+/]{1}===))$/;
```

Comment:
Nice, but isn't padding technically optional in base64?

Reply:
Depends who you ask. The strict and original version requires padding. But there are other flavours and specs around.